cpp/vcpkg.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can a comment be added to summarise what it's form?

Reply:
No comments allowed in json but could put in readme.